### PR TITLE
Change README title

### PR DESCRIPTION
**Pull request by djKooks**
Change README title, as module name changed

### CODE REVIEW

**Comment by djKooks** [comment]
I brought phrase `A framework for creating HTTP/HTTPS server` from the README file in root. Will it be better to describe `framework` to `module`, or is `framework` correct?

File: http/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# net
-
-Usage:
+# http
+A framework for creating HTTP/HTTPS server.
```
Comment (zhmushan) [inline http/README.md]: I prefer `module`

File: http/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# net
-
-Usage:
+# http
```
Comment (zhmushan) [inline http/README.md]: You need to run format.ts

Comment (djKooks) [reply http/README.md]: @zhmushan thanks! Seems `format.ts` is not effecting `md` files inside module, so updated it too.

**Pull request merged**